framework/test/driver/LDAPFilterTest.cpp

```diff
@@ -46,6 +46,10 @@ int TestParsing()
                       "=Person)(|(sn=Jensen)(cn=Babs J*)))");
     US_TEST_OUTPUT(<< "Parsing (o=univ*of*mich*)")
     ldap = LDAPFilter("(o=univ*of*mich*)");
+    US_TEST_OUTPUT(<< "Parsing (prop=foo(bar))")
```

Comment:
there should also be at least one test case that tests invalid (i.e. non-matching parenthesis) input

Reply:
adding more test cases